frontend/packages/employee-frontend/src/types/invoicing.ts

```diff
@@ -265,6 +266,7 @@ export interface FeeDecisionDetailed {
   approvedAt: Date | null
   createdAt: Date
   sentAt: Date | null
+  financeDecisionManager: { employee: Employee } | null
```

Comment:
I don't think this field needs to be this detailed, something like `financeDecisionManager: string` with the value being the name would be enough.